Misc/NEWS.d/next/Core_and_Builtins/2024-10-27-22-23-57.gh-issue-125873.BG97KU.rst

```diff
@@ -1 +1 @@
-Improved error message when ``PYTHONHOME`` is invalid.
+Improved error message when an environment variable is misonfigured.
```

Comment:
```suggestion
Improved error message when an environment variable is misconfigured.
```

Reply:
Thanks, already addressed this in https://github.com/python/cpython/pull/126027/commits/c1b0264becfe8dc320c33c6b0c4b025dd2e5f17c